frameworks/helloworld/tests/test_soak.py

```diff
@@ -1,11 +1,91 @@
 import pytest
+from shakedown import *
```

Comment:
What is the pattern you would prefer to see?

Also, I will die before we do strict pep8 linting on our test code.